services/scabbard/libscabbard/Cargo.toml

```diff
@@ -84,7 +84,7 @@ experimental = [
 authorization = ["splinter/authorization"]
 client = []
 client-reqwest = ["client", "reqwest"]
-diesel-receipt-store = ["diesel", "sawtooth/transaction-receipt-store-lmdb"]
+diesel-receipt-store = ["diesel", "sawtooth/lmdb"]
```

Comment:
Should this go under the lmdb feature?

Reply:
Renamed "diesel-receipt-store" to "receipt-store" because we put both the diesel backed and lmdb backed receipt store impls behind the feature